tac/governing-documents/repository-structure.md

```diff
@@ -3,7 +3,7 @@ layout: default
 title: Common Repository Structure
 parent: Governing Documents
 grand_parent: LF Decentralized Trust TAC
-nav_order: 1
```

Comment:
I'm a little confused, there were two number 1's prior to the change, and zero number 1's after the change. Maybe I'm missing something...

Reply:
1 is used for the charter in https://github.com/LF-Decentralized-Trust/governance/pull/30